packages/exportable-manifest/src/index.ts

```diff
@@ -68,11 +68,19 @@ async function makePublishDependencies (dir: string, dependencies: Dependencies
   return publishDependencies
 }
 
+/** Check if a workspace dependency specifier matches the provided alias */
+function matchesWorkspaceAlias (depSpec: string, alias: '*' | '^' | '~'): boolean {
+  return depSpec === `workspace:${alias}` || depSpec.endsWith(`@${alias}`)
```

Comment:
it is not an alias that you check. It is the version prefix.

Reply:
I thought about `workspace:*`, `workspace:^` and `workspace:~` as **version** aliases, as opposed to package name aliases. The reason for the term "alias" is they are not semver specifications (`workspace:*` has completely different meaning than semver `*`) but with some context information they can be mapped to a semver/npm-alias specification.

Other alternative terms that come to my mind are "dynamic specification", "spec variable" or just making a strong distinction between pkgAlias and specAlias.